hasher-matcher-actioner/scripts/set_threatexchange_source

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+# Use Threatexchange from pypi or from a directory. If python-threatexchange
+# from the parent directory is copied over as local-threatexchange, it causes
+# the docker build to use it.
+
+# Since local-threatexchange is installed _after_ pip install, it will override
+# threatexchange.
+
+LOCAL_LIB_LOCATION='./local-threatexchange'
+
+usage() {
+    printf 'Usage: %s pypi | local \n' "$(basename "$0")"
+}
+
+if [ ! -d "hmalib" ]; then
+    printf 'Run from hasher-matcher-actioner directory\n'
+    exit 1
+fi
+
+if [ $# -eq 0 ]; then
+    usage && exit 1
+fi
+
+if [ $1 == "pypi" ]; then
+    rm -r $LOCAL_LIB_LOCATION 2>/dev/null
+    pip install . # reinstall from pypi
+    exit 0
+elif [ $1 == "local" ]; then
+    pip uninstall -y threatexchange # remove from current site-packages
+
+    # Copy and install from source. Copying is necessary. A link would not work.
+    # Dockerbuilds can't follow symlinks outside their context directory.
+    rm -r $LOCAL_LIB_LOCATION 2>/dev/null
+    cp -r ../python-threatexchange/ $LOCAL_LIB_LOCATION
```

Comment:
Symlink doesn't work on windows, but is it an option?

Reply:
It is "not" an option. Symlinks outside of docker context do not work.

When you do a `docker build`, you'll notice docker sends the entire context (the working directory) to the docker daemon. Symlinks are not traversed.